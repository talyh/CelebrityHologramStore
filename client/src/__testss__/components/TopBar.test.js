/*
* TESTS
* 
* Props and Rendering (a few can be covered by snapshot)
* 
* 1. Header.id has header
* 
* 
* 2. AppTitle.id has title
* 
* 3. AppTitle.onClick has props.onTitleClick
* 
* 4. AppTitle.children has props.title
* 
* 
* 5. AppSubtitle.id has subtitle
* 
* 6. AppSubtitle.children has props.subtitle
* 
* 
* 7. SearchBar.onSearch has props.onSearch
*/