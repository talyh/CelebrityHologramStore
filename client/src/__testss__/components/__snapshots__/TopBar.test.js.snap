// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit Tests Rendering matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TopBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <styled.h1
          id="title"
          onClick={undefined}
/>,
        <styled.h2
          id="subtitle"
/>,
        <SearchBar
          onSearch={undefined}
/>,
      ],
      "id": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": undefined,
          "id": "title",
          "onClick": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": undefined,
          "id": "subtitle",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "onSearch": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <styled.h1
            id="title"
            onClick={undefined}
/>,
          <styled.h2
            id="subtitle"
/>,
          <SearchBar
            onSearch={undefined}
/>,
        ],
        "id": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": undefined,
            "id": "title",
            "onClick": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": undefined,
            "id": "subtitle",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "onSearch": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
